release-notes/opensearch-ml-common.release-notes-2.17.0.0.md

```diff
@@ -0,0 +1,38 @@
+## Version 2.17.0.0 Release Notes
+
+Compatible with OpenSearch 2.17.0
+
+### Features
+* Offline batch ingestion API actions and data ingesters (#2844)[https://github.com/opensearch-project/ml-commons/pull/2844]
+* Support get batch transform job status in get task API (#2825)[https://github.com/opensearch-project/ml-commons/pull/2825]
+
+### Enhancements
+* Adding additional info for memory metadata (#2750)[https://github.com/opensearch-project/ml-commons/pull/2750]
+* Support list in response body (#2811)[https://github.com/opensearch-project/ml-commons/pull/2811]
```

Comment:
this commit get reverted. Instead, can you add this enhancement? 
* Fix custom prompt substitute with List issue (#2871)[https://github.com/opensearch-project/ml-commons/pull/2871]

Reply:
removed.